corebc-blockindex/src/utils.rs

```diff
@@ -22,16 +21,13 @@ pub enum YlemLookupResult {
 
 /// Returns the requested Ylem version(s).
 pub async fn lookup_compiler_version(query: &YlemLookupQuery) -> Result<YlemLookupResult> {
```

Comment:
Function looks good, but would be nice to have some simple test cases to make sure it is working properly

Reply:
There are 4 tests for this function:
can_lookup_compiler_version_latest
can_lookup_compiler_version_exact
can_lookup_compiler_version_all
can_lookup_compiler_version_exact_wrong

https://github.com/core-coin/corebc-rs/pull/5/files#diff-e8ce521bb90f8ae7a40af64efb88ca91ccb6ca836ff84ac7af8e82a574a1b72eR6